src/sentry/replays/blueprints/api.md

```diff
@@ -608,3 +608,107 @@ Retrieve a collection of click events associated with a replay.
     ]
   }
   ```
+
+## Replays Viewed [/organizations/<organization_slug>/replays-viewed/]
+
+- Parameters
+
+  - project (optional, string)
+  - id (required, array[string]) - The id of the replay to retrieve. Can be specified multiple times.
```

Comment:
Change to "ids" to emphasize this is plural

Reply:
What's meant by "specify multiple times"?

Reply:
Is there any difference between this array and the one returned by the endpoint? Does the endpoint maybe validate + check for existence of ids?

Reply:
The parameter is specified like this: `?id=a&id=b`.

Reply:
No, no need to validate.  We just want to know if the replay-id exists in the set or not.